dev/source/docs/ros-distance-sensors.rst

```diff
@@ -0,0 +1,72 @@
+.. _ros-distance-sensors:
+
+==========================
+ROS distance sensors usage
+==========================
+
+.. warning::
+
+    Distance sensors message are currently support on :
```

Comment:
The message is named DISTANCE_SENSOR so I think using singular here would be better.